### ISSUE TITLE
Help text not visible

### ISSUE THREAD

**Issue by bunnybot** [widelands/widelands#6926]
[<img align='left' src='https://codeberg.org/avatars/e5346b63953ea3d8a9250d30c74606f6f8029a54e0b6bb6148e52b2c19594709' alt='janus' height='48' />](https://codeberg.org/janus)*[Mirrored from Codeberg](https://codeberg.org/wl/widelands/issues/5286)*
*Created on Mon Dec 08 21:21:05 CET 2025 by [**janus**](https://codeberg.org/janus)*

---

### Describe the Bug

When working with a larger mouse pointer in order to find it quickly on large or multiple screens, the help for an object is only partially displayed when hovering over it with the mouse. In the game, there is the option not to use the system mouse pointer size, but then the mouse jerks as described in the help section.

### To Reproduce

1. increase mouse curser size
2. start widelands
3. mouseover a item
4. help text not fully visible

### Expected Behavior

Even with a larger mouse pointer, the help should not be obscured.

### Screenshots

These two images should make it clear.

### Operating System

MacOS Big Sur, Windows 10 22H2

### Widelands Version

1.3 git b280568 release

### Enabled Add-Ons

no one
